Migração de versão das Assinaturas (#1474)
mercadopago/devsite-docs: guides/online-payments/subscriptions/migration_v0.en.md

# Migration from V1 to V0

As of March 8, 2022 a new version of the signature engine related to the entities below will be available with new features.

* **Plan** (identified by /v1/plan in v1 and now by `preapproval_plan` in v0)
* **Subscription** (identified by /v1/subscriptions in v1 and now by `preapproval` in v0)
* **Invoice** (identified by /v1/invoices in v1 and now by `authorized_payments` in v0)

The new version has a superior architecture with much more functionality and has email notifications sent with each status update at:

* Subscriptions (creation and cancellation);
* Subscription charge;
* Change of plans.

In addition to email notifications, the new version also has [Webhooks](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/en/guides/notifications/webhooks) for `plans`, `subscriptions` and `invoices`.

> WARNING
>
> Important
>
> Unlike Webhook notifications, which you need to set up, sending emails is automatically enabled by default. However, you can request manual shutdown of this functionality through Integrations.

## Migration

> You will need to migrate to this new engine because version 1 **will also cease to be available on March 8, 2022**.

Reviewer: Heitor, aqui sugiero un cambio de estructura de la oración (opcional, tu decision). Creo que quedaria bien poner: Migration to this engine is required, as **Version 1 will no longer be available from March 8, 2022 onwards**.

The migration will happen automatically and all migrated plans, subscriptions and invoices will remain active with a new ID. To verify the new migrated IDs, you can use the **endpoint** below which will remain public at your disposal until July 2022.

> With the new ID in hand, you can update your plan IDs, subscriptions and invoices as you wish.

Reviewer: ```suggestion
Once you have the new ID, you can update your plan IDs, subscriptions and invoices as you wish.
```
